modules/core/04-channel/keeper/multihop_packet_test.go

```diff
@@ -103,13 +341,19 @@ func (suite *MultihopTestSuite) TestRecvPacket() {
 					suite.Require().False(receiptStored, "packet receipt stored in ORDERED channel")
 				} else {
 					suite.Require().True(channelZ.Ordering == types.UNORDERED)
-					suite.Require().Equal(nextSeqRecv, packet.GetSequence(), "sequence incremented in UNORDERED channel")
-					suite.Require().Equal(nextSeqRecv, uint64(2), "sequence incremented in UNORDERED channel")
+					// suite.Require().Equal(nextSeqRecv, packet.GetSequence(), "sequence incremented in UNORDERED channel")
```

Comment:
Is this one meant to be commented?

Reply:
If I un-comment it the third success test case will fail. I am not sure if it is an invalid equality requirement, or a real issue that needs to be addressed.

Reply:
Looked at this again just now. It looks like maybe you added this check? The single hop test does not have this check and the reason is because the packet sequence ordering is only important for ORDERED channels (think TCP). For UNORDERED channels (UDP) packets are not required to be delivered in order.

So the line can be removed.

Reply:
Removed this line.